terraform/modules/application/ecs.tf

```diff
@@ -2,6 +2,11 @@
 resource "aws_ecs_cluster" "this" {
   #checkov:skip=CKV_AWS_65:TODO CLDC-2542 enable container insights if necessary for logging/monitoring
   name = var.prefix
+
+  setting {
+    name  = "containerInsights"
+    value = "enabled"
+  }

```

Comment:
FYI: Container insights is relatively expensive, because it sends a number of metrics per cluster / per task family / per service to cloudwatch which are all charged as custom metrics (the basic ones automatically sent by AWS services are not charged for). This is why AWS tells you about the workaround using a count on a CPU metric. Looking at it at the moment on staging it's sending 63 metrics, which I think works out at ~$18.90 per month.

We're probably ok with that (certainly it's dwarfed by the cost of dev time), but it's worth being aware of with this kind of thing.

Reply:
@RachaelBooth I've switched to the cpu count approach to save money - only took me 10 mins so worth the time I think! Lmk if you're happy with the new metric :) Thanks!

Reply:
LGTM